src/ext/file.rs

```diff
@@ -266,7 +266,7 @@ pub trait FileExt: AsRawHandle {
     }
 
     /// Gets various characteristics of a placeholder using the passed blob size.
-    fn sync_root_info_unchecked(&self, blob_size: usize) -> core::Result<SyncRootInfo> {
+    unsafe fn sync_root_info_unchecked(&self, blob_size: usize) -> core::Result<SyncRootInfo> {
```

Comment:
Since this function is marked unsafe we can remove the unsafe blocks inside.

Reply:
<https://rust-lang.github.io/rfcs/2585-unsafe-block-in-unsafe-fn.html>